common/buildcraft/compat/CompatModuleForestry.java

```diff
@@ -1,29 +1,56 @@
 package buildcraft.compat;
 
-import forestry.storage.*;
-import forestry.api.storage.*;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+
+import net.minecraftforge.client.MinecraftForgeClient;
+
+import cpw.mods.fml.common.FMLCommonHandler;
+import cpw.mods.fml.common.event.FMLMissingMappingsEvent;
+import cpw.mods.fml.common.registry.GameRegistry;
+
+import forestry.api.core.ForestryAPI;
+
+import buildcraft.BuildCraftTransport;
+import buildcraft.compat.forestry.pipes.PipeItemsPropolis;
+import buildcraft.core.BCCreativeTab;
+import buildcraft.transport.BlockGenericPipe;
+import buildcraft.transport.TransportProxyClient;
 
 public class CompatModuleForestry extends CompatModuleBase
 {
-    private static BackpackDefinition backpackPlumber;
-    private static BackpackDefinition backpackArchitect;
-    private static BackpackDefinition backpackPipeEngineer;
-    private static BackpackDefinition backpackRobotEngineer;
+    /** Pipe used to sort bees from Forestry. */
+    private static Item pipeItemsPropolis;
     
     @Override
     public String name() {
         return "Forestry";
     }
-    
+
+    @Override
+    public boolean canLoad() {
+        return super.canLoad() && ForestryAPI.enabledPlugins.contains("APICULTURE");
+    }
+
     @Override
     public void init() {
-        BackpackManager.backpackInterface.addBackpack((IBackpackDefinition)CompatModuleForestry.backpackPlumber, EnumBackpackType.T1);
```

Comment:
Someone has a branch with Backpack stuff but it's all done in ForestryInit, so I took out the backpack stuff here.